packages/components/src/tools-panel/stories/index.js

```diff
@@ -79,6 +83,54 @@ export const _default = () => {
 	);
 };
 
+export const WithOptionalItemsPlusIcon = () => {
+	const [ height, setHeight ] = useState();
+	const [ width, setWidth ] = useState();
+
+	const resetAll = () => {
+		setHeight( undefined );
+		setWidth( undefined );
+	};
+
+	return (
+		<PanelWrapperView>
+			<Panel>
+				<ToolsPanel
+					label="Tools Panel (with optional items only)"
```

Comment:
The current length of the label forces it to wrap so the menu placement looks odd. This little tweak means we can leave the panel width consistent with how it's used in the editor etc. What do you think?

```suggestion
					label="Tools Panel (optional items only)"
```

| Before | After |
|---|---|
| <img width="275" alt="Screen Shot 2021-10-13 at 4 15 23 pm" src="https://user-images.githubusercontent.com/60436221/137077950-480e2e9c-f1b9-41ce-af19-6694344c5f57.png"> | <img width="273" alt="Screen Shot 2021-10-13 at 4 15 06 pm" src="https://user-images.githubusercontent.com/60436221/137077973-89308c5d-9fcd-46e2-902a-9be5549532a6.png"> |